From f14b0079cc7b8d5a6cd6581630902359be605a80 Mon Sep 17 00:00:00 2001
From: Yotam BenDosa <ybendosa@habana.ai>
Date: Wed, 08 Apr 2020 16:11:32 +0300
Subject: [PATCH] Revert "[SW-0] openbmc: install-hls: fix part verification"

This reverts commit 4ceebe329ee7d0611b3b480fd6f5fd1bba5a0b9e.

It seems that unlike other partition images, the u-boot partition image
size is only dependent on the size of the product of its build process,
and not on the definition provided in the openbmc layer.

This means that it doesn't align to the kernel dts definition, and will
fail partitions verification unnecessarily.

Change-Id: I0d276a1d9e73c72e7018d1442a5ae6b377fa7b39
---

diff --git a/package/openbmc/install-hls-curr.sh b/package/openbmc/install-hls-curr.sh
index 9de46c4..2bd655a 100755
--- a/package/openbmc/install-hls-curr.sh
+++ b/package/openbmc/install-hls-curr.sh
@@ -146,7 +146,7 @@
 	local mtd_size=$((16#${mtd_hex_size}))
 	local img_size=$(tar -vtf ${BMC_IMAGE} ${partition_image} | cut -d ' ' -f3)
 
-	if [ ${img_size} -ne ${mtd_size} ]; then
+	if [ ${img_size} -gt ${mtd_size} ]; then
 		echo 0
 	else
 		echo 1
